app/Notifications/NewBon.php

```diff
@@ -42,6 +43,24 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     */
+    public function shouldSend(User $notifiable): bool
+    {
```

Comment:
Should have an is_modo check check to force send.

Reply:
Added